src/network/networkd-network-gperf.gperf

```diff
@@ -243,6 +243,7 @@ BridgeVLAN.PVID,                             config_parse_brvlan_pvid,
 BridgeVLAN.VLAN,                             config_parse_brvlan_vlan,                                 0,                             0
 BridgeVLAN.EgressUntagged,                   config_parse_brvlan_untagged,                             0,                             0
 Network.IPv6PrefixDelegation,                config_parse_router_prefix_delegation,                    0,                             0
+Network.IPv6PDSubnetId,                      config_parse_int32,                                       0,                             offsetof(Network, router_prefix_subnet_id)
```

Comment:
As explained above, I'd find it more natural to pass it in hex notation - might need a special parsing function.